TestCases/nonequilibrium/thermalbath/finitechemistry/weakly_ionized.cfg

```diff
@@ -0,0 +1,92 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%                                                                              %
+% SU2 configuration file                                                       %
+% Case description: 2D 5x5 Square Test Case representing a thermal bath        %
+%                    with thermal nonequilibrium and finite-rate chemistry     %
+%                                                                              %
+%                   This is only serves as a validation case.                  %
+%                                                                              %
+%                                                                              %
+% Author: W. Maier                                                             %
+% File Version 7.3.0 "Blackbird"                                               %
+%                                                                              %
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+% ------------- DIRECT, ADJOINT, AND LINEARIZED PROBLEM DEFINITION ------------%
+%
+% Physical governing equations (EULER, NAVIER_STOKES,
+%                               WAVE_EQUATION, HEAT_EQUATION, FEM_ELASTICITY,
+%                               POISSON_EQUATION)
+SOLVER= NEMO_EULER         
+GAS_MODEL= AIR-7
+GAS_COMPOSITION=(0.0, 0.767, 0.233, 0.0, 0.0, 0.0, 0.0)
+MATH_PROBLEM= DIRECT
+READ_BINARY_RESTART= NO
+
+TIME_MARCHING=TIME_STEPPING
```

Comment:
You could use this to cover the dual-time fix.

Reply:
👍